sphinxcontrib/inmanta/api.py

```diff
@@ -420,9 +568,14 @@ def get_module_filter(self, module_folder: Optional[str]) -> Callable[[str], boo
         if not os.path.exists(pyproject):
             return lambda x: True
 
-        with open(pyproject, "r") as fh:
+        with open(pyproject, "r"):
```

Comment:
I'm not sure if we even need to open the file. This feels weird to me. I would expect we'd either pass the handle, or leave it to `toml` to read the file.